Economy.md

```diff
@@ -0,0 +1,89 @@
+# EssentialsX Economy
+EssentialsX comes with a simple, easy-to-use economy system. The built in economy API will redirect to any other detected economy system such as vault. This means you can use EssentialsX Economy as a economy API instead of Vault if your plugin is dependent on EssentialsX.
```

Comment:
```suggestion
EssentialsX comes with a simple, easy-to-use economy system. The built-in economy API will redirect to any other detected economy system such as Vault. This means you can use EssentialsX Economy as a economy API instead of Vault if your plugin is dependent on EssentialsX.
```

IMO the economy API doesn't belong in the opening paragraph. The first paragraph should ideally explain EssentialsX's economy to server admins, but the economy API is only useful for plugin developers. We could have a separate section about Vault and plugin developers further down.

Reply:
Oh, I forgot to move that :P